gc/base/HeapMap.hpp

```diff
@@ -255,6 +256,30 @@ class MM_HeapMap : public MM_BaseVirtual
 		return false;
 	}
 	
+	MMINLINE uintptr_t *
+	getSlotPtr(uintptr_t index)
+	{
+		return &(getHeapMapBits()[index]);
+	}
+
+	MMINLINE const uintptr_t *
+	getSlotPtr(uintptr_t index) const
+	{
+		return &(getHeapMapBits()[index]);
+	}
+
+	MMINLINE uintptr_t *
+	getSlotPtrForAddress(omrobjectptr_t address)
+	{
+		return getSlotPtr(getSlotIndex(address));
+	}
```

Comment:
don't understand why these non-const variants are necessary!?

Reply:
To me, having both seems like a more complete APIs, but that's only a slight preference. If you don't want them, I don't mind. There are places where we could replace `&(_heapMapBits[index])` with the non-const variant of `getSlotPtr`, for example, `setBit`. I don't know if there's any that could use the non-const `getSlotPtrForAddress`.

Reply:
no strong preference either. I'll leave it up to you.

Reply:
OK, we will leave them in then :)